scripts/publish_cft.sh

```diff
@@ -15,10 +15,32 @@ function upload_file() {
 }
 
 version=$(grep defaultBeatVersion version/version.go | cut -f2 -d "\"")
-upload_file deploy/cloudformation/elastic-agent-ec2-cnvm.yml "cloudformation-cnvm" "$version"
-upload_file deploy/cloudformation/elastic-agent-ec2-cspm.yml "cloudformation-cspm-single-account" "$version"
-upload_file deploy/cloudformation/elastic-agent-ec2-cspm-organization.yml "cloudformation-cspm-organization-account" "$version"
-upload_file deploy/cloudformation/elastic-agent-direct-access-key-cspm.yml "cloudformation-cspm-direct-access-key-single-account" "$version"
-upload_file deploy/cloudformation/elastic-agent-direct-access-key-cspm-organization.yml "cloudformation-cspm-direct-access-key-organization-account" "$version"
-upload_file deploy/asset-inventory-cloudformation/elastic-agent-ec2.yml "cloudformation-asset-inventory-single-account" "$version"
-upload_file deploy/asset-inventory-cloudformation/elastic-agent-ec2-organization.yml "cloudformation-asset-inventory-organization-account" "$version"
+
+upload_file deploy/cloudformation/elastic-agent-ec2-cnvm.yml \
+    "cloudformation-cnvm" \
+    "${version}"
+upload_file deploy/cloudformation/elastic-agent-ec2-cspm.yml \
+    "cloudformation-cspm-single-account" \
+    "${version}"
+upload_file deploy/cloudformation/elastic-agent-ec2-cspm-organization.yml \
+    "cloudformation-cspm-organization-account" \
+    "${version}"
+upload_file deploy/cloudformation/elastic-agent-direct-access-key-cspm.yml \
+    "cloudformation-cspm-direct-access-key-single-account" \
+    "${version}"
+upload_file deploy/cloudformation/elastic-agent-direct-access-key-cspm-organization.yml \
+    "cloudformation-cspm-direct-access-key-organization-account" \
+    "${version}"
+upload_file deploy/cloudformation/cloud-connectors-remote-role.yml \
+    "cloudformation-cloud-connectors-single-account" \
+    "${version}"
+upload_file deploy/cloudformation/cloud-connectors-remote-role-organization.yml \
+    "cloudformation-cloud-connectors-organization-account" \
+    "${version}"
+
```

Comment:
nitpick any specific reason to add an extra line break here?

Reply:
The intention was to separate cpsm from asset inventory.